lessons/introduction-to-smartpy/14/14.mdx

```diff
@@ -102,11 +102,11 @@ editor:
 
 ### Intro: 
 
-You would be already be pretty familiar with basic decision making in Python using `if...else` statements.  
+You should already be pretty familiar with basic decision making in Python using `if...else` statements.  
```

Comment:
@STFinancial "should" gives a strong connotation. I would let it "would be" remain as it is. 

Our principles for content design were (should be documented somewhere, thanks for bringing this up)
1. Take no assumptions for learner's state of mind and skills. 
2. Give as much affordance to the learner.

Reply:
That's fine, but "would be already be" says "be" twice and not grammatical, at least in my american english. Perhaps "you may already be pretty familiar" would be better instead?

Reply:
> That's fine, but "would be already be" says "be" twice and not grammatical
Agreed. 

> Perhaps "you may already be pretty familiar" would be better instead?
Agreed. Request to make this change and we can merge this 🚀